[2024-05-30T11:31:14.413+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-30T11:31:14.434+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.insert_uuid manual__2024-05-30T11:26:08.851759+00:00 [queued]>
[2024-05-30T11:31:14.442+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.insert_uuid manual__2024-05-30T11:26:08.851759+00:00 [queued]>
[2024-05-30T11:31:14.443+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-30T11:31:14.457+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): insert_uuid> on 2024-05-30 11:26:08.851759+00:00
[2024-05-30T11:31:14.464+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=192) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-30T11:31:14.466+0000] {standard_task_runner.py:63} INFO - Started process 194 to run task
[2024-05-30T11:31:14.466+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'insert_uuid', 'manual__2024-05-30T11:26:08.851759+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpg52vpjjm']
[2024-05-30T11:31:14.469+0000] {standard_task_runner.py:91} INFO - Job 238: Subtask insert_uuid
[2024-05-30T11:31:14.517+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.insert_uuid manual__2024-05-30T11:26:08.851759+00:00 [running]> on host 29c09f03cc50
[2024-05-30T11:31:14.600+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='insert_uuid' AIRFLOW_CTX_EXECUTION_DATE='2024-05-30T11:26:08.851759+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-30T11:26:08.851759+00:00'
[2024-05-30T11:31:14.602+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-30T11:31:14.620+0000] {logging_mixin.py:188} INFO - UUID inserted successfully!
[2024-05-30T11:31:14.620+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-30T11:31:14.621+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-30T11:31:14.631+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=insert_uuid, execution_date=20240530T112608, start_date=20240530T113114, end_date=20240530T113114
[2024-05-30T11:31:14.681+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-30T11:31:14.711+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-30T11:31:14.712+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
